api.js

```diff
@@ -137,6 +144,11 @@ Api.prototype.run = function () {
 				return Promise.reject(new AvaError('Couldn\'t find any files to test'));
 			}
 
+			var cache = self.options.cache !== false;
```

Comment:
I'd use `isCacheEnabled`, something more verbose & logical given the boolean variable value.